### ISSUE TITLE
Add Chinese version

### ISSUE THREAD

**Issue by huanglin-github** [KS-DB-Merge-Tools/Issues#7]
Add Chinese version

**Comment by semenenkov** [comment]
Hello,

Thanks a lot for the suggestion, unfortunately I have to close it as "won't do" for now. App UI translation is a huge task that should be constantly supported and it will slow down the further development.

**Comment by semenenkov** [comment]
@huanglin-github I made some changes to make the app a bit more easy to translate:
- Installer - Help button on the Welcome screen, opening the new Installation help page which contains texts from all installation steps
- Documenation - Lot of pages were updated to include the text from app UI, the most important I suppose are the dialog docs, dialogs can contain lot of text
- App - Basic yes/no and ok/cancel dialogs which have no separate help pages - they now have a Copy button in the dialog title bar, to make it easier to translate the dialog text
- App - The starting Activation dialog includes new "Interface language and documentation" section providing details about these possibilities

To my understanding, the only thing which still can be difficult to translate are the toolbar button tooltips. Under certain conditions they can include action deny reason, and there can be lot of different reasons for each action. It requires lot of time to analyze and describe this logic in the documentation. I've created a task for myself to do this, but I don't know when I will have time for it.
